libstorage/MemoryTable2.cpp

```diff
@@ -43,6 +43,37 @@ Entries::ConstPtr MemoryTable2::select(const std::string& key, Condition::Ptr co
     return selectNoLock(key, condition);
 }
 
+void MemoryTable2::proccessLimit(const Condition::Ptr& condition,
+    const std::shared_ptr<dev::storage::Entries>& entries, const Entries::Ptr& resultEntries)
+{
+    if (condition->getOffset() >= 0 && condition->getCount() >= 0)
+    {
+        std::vector<size_t> limitedIndex;
+        limitedIndex.reserve(entries->size());
+        for (size_t i = 0; i < entries->size(); ++i)
```

Comment:
This can be simplify. you don't need index any more. Just add asked number of entry's point to entries.

Reply:
recontruct the code

Reply:
reconstruct the code